app/src/main/java/com/example/concordia_campus_guide/Models/BusStop.java

```diff
@@ -0,0 +1,22 @@
+package com.example.concordia_campus_guide.Models;
+
+import com.example.concordia_campus_guide.ClassConstants;
+
+public class BusStop extends Place {
+
+    public BusStop(String campus) {
+        if (campus.equals("SGW")) {
```

Comment:
maybe add it to the constants class? or is it overkill?

Reply:
Since the coordinates are in the classCostants I think this is fine here